### ISSUE TITLE
Hypoplastic anterior limbs of the internal capsules

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7171]
**Preferred term label:**
Hypoplastic anterior limbs of the internal capsules

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615673 
PMID: 32395406  1/1

**Your nano-attribution (ORCID)**
